### PR TITLE
A bunch of small improvements

### LINKED ISSUE #202: Regarding transaction fees in parachains
Transaction fees can also be an implementation choice of parachains. Unlike Cosmos, Polkadot makes no hard and fast rules for how the parachains decide on original validity of transctions. For example, a parachain may be implemented so that transactions must pay a minimum fee to collators to be valid. The relay chain will enforce this validity. Similarly, a parachain could not include that in their implementation and Polkadot would still enforce its validity.

### PR DESCRIPTION
**Pull request by lsaether**
closes #202

### CODE REVIEW

**Comment by laboon** [comment]
Logan - sorry, I totally missed this when it came in.  Reviewing now.

File: docs/polkadot/learn/architecture.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Architecture
```
Comment (laboon) [inline docs/polkadot/learn/architecture.md:1]: There doesn't seem be a link to the Architecture page on the sidebar.

**Comment by laboon** [comment]
Overall looks good with one minor comment in review.

**Review by laboon — CHANGES_REQUESTED** [review]

File: docs/polkadot/learn/architecture.md (hunk 1 above)
Comment (lsaether) [reply docs/polkadot/learn/architecture.md:1]: added

**Review by laboon — APPROVED** [review]
👍

**Pull request merged**